wagtailio/blog/templates/blog/blog_index_page.html

```diff
@@ -16,7 +16,8 @@ <h1 aria-hidden="true">Latest Chirps</h1>
 
         <section class="blog-listing">
             {% for post in posts %}
-                {% if forloop.first or forloop.counter|divisibleby:6 %}
+                {# only show featured posts on the first pagination page, no filters #}
+                {% if posts.number == 1 and post in featured_posts and 'category' not in request.GET %}
```

Comment:
question: the designs show featured items every few posts.
looking at this, it just loops through all blog posts (for the first page) and marks it as featured, if it is in featured posts..

This means that if the last 5 blog posts are also marked as featured, they all appear as such. Worth double checking with @vossisboss, but in my head we should have a separation between latest blog posts and featured blog posts, even if it may happen that the featured ones duplicate the latest ones on the first page

Reply:
I think @zerolab is right @engineervix. Featured blog posts will likely be some of our most recent posts, but not always. Sometimes there will be older content we'll want to highlight. I'm fine with the featured ones duplicating the latest ones on the page because the designs are visually distinct. If they all looked the same, then it would be more of an issue. Let me know if that helps @engineervix.

Reply:
@engineervix this still need addressing, and then we'd be good to go

Reply:
Thanks @zerolab and @vossisboss, this has been addressed in acfe1c7

With this approach, we'll have duplicate entries on the first page for any of the latest posts which are featured.